rustls/examples/internal/bogo_shim.rs

```diff
@@ -9,6 +9,7 @@ use env_logger;
 use rustls;
 
 use rustls::internal::msgs::codec::Codec;
+use rustls::internal::msgs::enums::KeyUpdateRequest;
```

Comment:
This seems unrelated?

Reply:
This is currently used in the error handling shim further down in the file:
```rust
Error::CorruptMessage(InvalidMessage::InvalidKeyUpdate(KeyUpdateRequest::Unknown(42))) => {
            quit(":BAD_HANDSHAKE_MSG:")
        }
```

Would you prefer that I removed it in favor of a less explicit `InvalidMessage::InvalidKeyUpdate(_)` pattern instead?